src/app/ui/modal/common/confirm.component.ts

```diff
@@ -22,18 +27,27 @@ export class ConfirmComponent {
     public content: string;
 
     @Input()
+    public showDiscardButton: string;
```

Comment:
These new discard options are a bit specific to a single use. Maybe create a separate modal for closing dirty apps?